src/lib/mapping/sign.ts

```diff
@@ -1,19 +1,27 @@
 import { hash } from "../utils/arweaveUtils";
 import { bufferTob64Url } from "../utils/arweaveUtils";
 import { signature as getSignedData } from "./signature";
-import { getActivePublicKey } from "./getActivePublicKey";
+import type Transaction from "arweave/web/lib/transaction";
+import { getCachedUserPublicKey } from "../auth/auth0";
+
+// TODO: Rename to signTransaction
```

Comment:
Is it fine to rename methods (breaking change)? Should I keep an alias with the old name and mark it as deprecated?